connect-examples/v2/node_orders-payments/bin/script/seed-catalog.js

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 /* eslint no-console: 0 */
 
 const SquareConnect = require("square-connect");
-const config = require("../../config.json")["sandbox"];
+const config = require("../../config.json")[process.env.NODE_ENV];
```

Comment:
We probably wouldn't want to seed the production catalog with dummy items. We should make it so that this script only works with sandbox

Reply:
Good catch, I'll correct that.